src/Components/Form/SubmissionForm.tsx

```diff
@@ -79,17 +102,35 @@ const SubmissionForm: React.FC<MedProps> = ({ chosenMedicine }) => {
 
             </div>
             <div className="dosage-section">
-                <label htmlFor="dosage-num">Dosage: </label>
+                <label htmlFor="dosage-num">Single Dose :</label>
```

Comment:
Checked a few other labels, would recommend keeping ':' in same placement throughout. 

`Single Dose:`